waku/v2/protocol/waku_noise/noise.nim

```diff
@@ -62,6 +77,94 @@ type
     nonce: ChaChaPolyNonce
     ad: seq[byte]
 
+  #################################
+  # Noise handshake patterns
+  #################################
+
+  # The Noise tokens appearing in Noise (pre)message patterns 
+  # as in http://www.noiseprotocol.org/noise.html#handshake-pattern-basics
+  NoiseTokens* = enum
```

Comment:
Assuming these types are exported `*` to be unit tested or used in other modules?

Reply:
Good catch! There is no need to export the tokens as long as new handshake patterns are defined in this module. Removed in https://github.com/status-im/nwaku/pull/933/commits/ed3eff8855f531451f5003c14bea9db237dfa4bb